services/repository/src/main/java/org/sagebionetworks/repo/web/UrlHelpers.java

```diff
@@ -232,6 +232,7 @@
 	public static final String COPY = "/copy";
 	public static final String FILE_HANDLES_COPY = FILE_HANDLES + COPY;
 	public static final String ENTITY_DATA_TYPE = ENTITY_ID+"/datatype";
+	public static final String ENTITY_BIND_JSON_SCHEMA = ENTITY_ID+"/bound/json/schema";
```

Comment:
Might want to consider simplifying the URL to just /entity/{id}/schema/binding or /entity/{id}/jsonschema/binding